meinberlin/assets/scss/components_user_facing/_project-tile.scss

```diff
@@ -13,20 +13,33 @@ a.project-tile {
         height: 100%;
     }
 
-    &--horizontal {
+    &--horizontal:not(.project-tile--map) {
         border-bottom: 1px solid $gray-lighter;
         padding-bottom: 1.5em;
         padding-top: 1.5em;
     }
 }
 
 .project-tile__image-wrapper {
-    flex: 1
+    flex: 1;
```

Comment:
wondering if we should make this container have fixed sizes? im having some issues with the aspect ratio 
![tiles](https://github.com/user-attachments/assets/3fc995c4-8614-4364-b41b-cc5880d19df9)

Reply:
![tiles2](https://github.com/user-attachments/assets/7f0375a0-948e-4755-b220-2d60b7949aea)

Reply:
@hom3mad3 i cant reproduce the error with the images at all unfortunately. I did some size and aspect-ratio changes based on some of marcelos feedback, maybe that also fixes it for you, plus the removal of the `width: auto`

Reply:
works now!